### ISSUE TITLE
The Launcher Wont Save The Settings

### ISSUE THREAD

**Issue by Gthbacc** [openmultiplayer/launcher#219]
I downloaded the latest portable version of the launcher and installed it on top of a clean, pre-installed version of sa. But every time I open the launcher, all my settings (path, language, favorite servers) reset and I have to select them again. Not sure what the issue is exactly.
Btw I wanted to thank you for this project. Looks very neat & clean. Wish it was around in 2016 lol.

**Comment by AmyrAhmady** [comment]
Can you try running it as administrator and see if it fails to save or not?

**Comment by Gthbacc** [comment]
Still wont save. I made sure both omp and gta_sa.exe were running as administrator.

**Comment by Gthbacc** [comment]
Update: apparently the issue was that there was dot in the path where I installed gta sa (my path was: "OPEN.MP\Rockstar Games\GTA San Andreas"). One I changed the name of "OPEN.MP" folder, the issue was gone.

**Comment by Gthbacc** [comment]
Alright so I did some more digging. After some testing, I think I know why this issue happened. I have a copy of SA that I used solely for singleplayer purposes (located at D:\Rockstar Games\GTA San Andreas). It seems that the launcher seaches for this address by default (As I remember from years ago, SAMP would only work if the game files were located at  a path named "Rockstar Games\GTA San Andreas") and ignores where I've installed my other copy of SA which I use for multiplayer purposes (located at D:\Open Multiplayer\Rockstar Games\GTA San Andreas). I think if in the next update, this issue gets addressed and the launcher just goes by the address that you give it in the first place, this problem will be gone.
Thank you.

**Comment by RedNeckSnailSpit** [comment]
I also appear to have the same issue.
I reinstalled the launcher to `D:\Games\openmp\omp-launcher.exe` with no success.
I tried running as admin with no success.
`omp-launcher.log` only contains the following lines:
```
[00:00:00.001] (b28) INFO   starting 12 workers
[00:00:00.001] (b28) INFO   Tokio runtime found; starting in existing Tokio runtime
```
I don't think that's too useful, but I could be wrong.
I can't think of anything else I could try.

My actual game is installed in `D:\Games\Grand Theft Auto San Andreas`

Due to using OneDrive on Windows 11, my Documents folder is in `C:\Users\doug4\OneDrive\Documents`, however `C:\Users\doug4\Documents` does still exist with nothing in it. I have found that `C:\Users\doug4\Documents` is read only and I can't seem to change that. Looking at your code though I don't think that will affect anything.
